applications/plugins/Geomagic/CMakeLists.txt

```diff
@@ -28,7 +29,7 @@ include_directories( ${OPENHAPTICS_INCLUDE_DIR})
 add_library(${PROJECT_NAME} SHARED ${HEADER_FILES} ${SOURCE_FILES} ${SCENES_FILES} ${README_FILES})
 set_target_properties(${PROJECT_NAME} PROPERTIES COMPILE_FLAGS "-DPLUGIN_DATA_DIR=\"${CMAKE_CURRENT_SOURCE_DIR}/\"")
 set_target_properties(${PROJECT_NAME} PROPERTIES COMPILE_FLAGS "-DSOFA_BUILD_GEOMAGIC")
-target_link_libraries(${PROJECT_NAME} SofaHelper SofaUserInteraction SofaHaptics ${OPENHAPTICS_LIBRARIES})
+target_link_libraries(${PROJECT_NAME} SofaHelper SofaUserInteraction SofaHaptics SofaOpenglVisual ${OPENHAPTICS_LIBRARIES})
```

Comment:
You made SofaOpenGLVisual required for the Geomagic plugin, intended ?

Reply:
Sadely Geomagic & other uses OglModel in their cpp code. 
So it just make obvious that there is a dependency.